### PR TITLE
Proofread Label Smoothing, Layer Freezing, Mixup, NoOpModel, Progressive Resizing, and RandAugment docstrings

### PR DESCRIPTION
**Pull request by mcneela**
Proofread Label Smoothing, Layer Freezing, Mixup, NoOpModel, Progressive Resizing, and RandAugment docstrings

### CODE REVIEW

File: composer/algorithms/mixup/mixup.py [hunk 1]
```diff
@@ -28,25 +28,25 @@ def mixup_batch(input: torch.Tensor,
...
+    axis (``dim=0``).
 
     The relative weight of the original ``input`` versus the permuted copy is
     defined by the ``mixing`` parameter. This parameter should be chosen
     from a ``Beta(alpha, alpha)`` distribution for some parameter ``alpha > 0``.
     Note that the same ``mixing`` is used for the whole batch.
 
     Args:
-        input (torch.Tensor): input tensor of shape ``(minibatch, ...)``, where
+        input (:class:`torch.Tensor`): input tensor of shape ``(minibatch, ...)``, where
```
Comment (hanlint) [inline composer/algorithms/mixup/mixup.py:39]: I dont think these are needed -- can you check the current docs @ docs.mosaicml.com to see if the Tensor is already linked?

Comment (coryMosaicML) [reply composer/algorithms/mixup/mixup.py:39]: It looks like `Tensor` is already linked to me.

File: composer/algorithms/mixup/mixup.py [hunk 2]
```diff
@@ -64,7 +64,9 @@ def mixup_batch(input: torch.Tensor,
             X = torch.randn(N, C, H, W)
             y = torch.randint(num_classes, size=(N,))
             X_mixed, y_perm, mixing = mixup_batch(
-                X, y, alpha=0.2)
+                X, y,
+                alpha=0.2
+            )
```
Comment (coryMosaicML) [inline composer/algorithms/mixup/mixup.py:69]: This looks a bit funny to me. Is this how we typically format things here?

**Review by coryMosaicML — APPROVED** [review]
LGTM aside from Hanlin's question.

File: composer/algorithms/mixup/mixup.py (hunk 2 above)
Comment (hanlint) [reply composer/algorithms/mixup/mixup.py:69]: yeah it's usually:
```
X, 
y,
alpha=0.2
```

Comment (mcneela) [reply composer/algorithms/mixup/mixup.py:70]: My bad, just fixed it.

File: composer/algorithms/mixup/mixup.py (hunk 1 above)
Comment (mcneela) [reply composer/algorithms/mixup/mixup.py:40]: @hanlint and @coryMosaicML I tried removing the :class: prefix and torch.Tensor was not linked upon rebuilding the documents. Am I missing something?

Comment (ravi-mosaicml) [reply composer/algorithms/mixup/mixup.py:40]: Hmm, strange. That's fine to remove it here; I'll have to fix something in the sphinx `conf.py` later.

**Comment by ravi-mosaicml** [comment]
> Hi @ravi-mosaicml I see you're working on getting this one ready, but I think #1073 is more up to date and would replace this PR. I went back in and redid some of these because Hanlin requested changes.

Ah cool, that's fine. I'll merge this PR in, and then merge #1073 on top of it.

File: composer/algorithms/label_smoothing/label_smoothing.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ def smooth_labels(logits: torch.Tensor, target: torch.Tensor, smoothing: float =
             Default: ``0.1``.
 
     Returns:
-        targets_smooth (torch.Tensor): The smoothed targets
+        torch.Tensor: The smoothed targets
```
Comment (ravi-mosaicml) [inline composer/algorithms/label_smoothing/label_smoothing.py:41]:
```suggestion
        torch.Tensor: The smoothed targets.
```

File: composer/algorithms/no_op_model/no_op_model.py [hunk 4]
```diff
@@ -64,7 +66,7 @@ def validate(self, batch: Batch) -> Tuple[Any, Any]:
 
 
 class NoOpModel(Algorithm):
-    """Runs on :attr:`Event.INIT` and replaces the model with a dummy model of type NoOpModelClass."""
+    """Runs on :attr:`Event.INIT` and replaces the model with a a dummy :class:`.NoOpModelClass` instance."""
```
Comment (ravi-mosaicml) [inline composer/algorithms/no_op_model/no_op_model.py:69]:
```suggestion
    """Runs on :attr:`Event.INIT` and replaces the model with a dummy :class:`.NoOpModelClass` instance."""
```
